pkg/backtest/exchange.go

```diff
@@ -375,12 +376,12 @@ func (e *Exchange) SubscribeMarketData(
 	for symbol := range loadedSymbols {
 		symbols = append(symbols, symbol)
 	}
+	symbols = append(symbols, "FXSUSDT")
```

Comment:
debugging code?

Reply:
I didn't really find a way yet to init the loadedSymbols from config

Reply:
nor did the backtest execute any trades so I assume the trade service is not properly connected to the backtest exchange. Do you have any ideas what I am missing here?

Reply:
You have to set it in the config yaml. check `pkg/bbgo/config.go` in the `Backtest` struct.
```yaml
backtest:
    symbols:
        - BTCUSDT
        - ETHUSDT
```

Reply:
also you need to set the `sync` section maybe.
```yaml
sync:
  userDataStream:
    trades: true
    filledOrders: true
  sessions:
    - binance
  symbols:
    - BTCUSDT
```

Reply:
gonna try that thanks